tools/run-wpt.mjs

```diff
@@ -43,17 +45,35 @@ if (
   process.exit(0);
 }
 
-const isLinux = process.platform === 'linux';
-const isMacOS = process.platform === 'darwin';
-
-// The path to the browser binary.
-const BROWSER_BIN =
-  process.env.BROWSER_BIN ||
-  (isLinux
-    ? '/usr/bin/google-chrome-unstable'
-    : isMacOS
-    ? '/Applications/Google Chrome Dev.app/Contents/MacOS/Google Chrome Dev'
-    : '');
+let BROWSER_BIN = process.env.BROWSER_BIN;
+if (!BROWSER_BIN) {
+  switch (process.platform) {
+    case 'linux':
+      BROWSER_BIN = join(sep, 'usr', 'bin', 'google-chrome-unstable');
```

Comment:
More readable, no?

      BROWSER_BIN = '/usr/bin/google-chrome-unstable';

Reply:
Same for macOS.

Reply:
It's just odd to use both `join` and `sep` at the same time. The fact you put `sep` at the beginning paramounts to hard-coding the fact Unix absolute paths start with `/`. So you might as well completely remove `sep`.

Reply:
This is more programmatic which is why I chose it. Since this is more versatile, I'll just keep it as it.